activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -107,7 +107,8 @@ def take!
     end
 
     # Find the first record (or first N records if a parameter is supplied).
-    # If no order is defined it will order by primary key.
+    # If no order is defined it will order by primary key if the collection is not loaded.
+    # If the collection is loaded it will get the first record from that collection.
```

Comment:
Maybe:

```
# If the collection is not already loaded in memory, it will order by primary key if no order is specified.
# When the collection is already in memory it will return the first record.
```